tests/positive/Internal/LiteralStringMain.juvix

```diff
@@ -0,0 +1,8 @@
+module LiteralStringMain;
+
+open import Stdlib.Prelude;
+
+main : ℕ;
+main ≔ 123;
```

Comment:
Weren't we supposed to change ≔to :=? This still works?

Reply:
This file is unused - I've removed it.